tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py33,py34,py35
+envlist = py27,py34,py35
```

Comment:
I know clound9 systems didn't come intalled with _py33_ but you can install that using `$ sudo apt-get install python3.3` and then `tox` will run perfectly.